content/guides/dam-connection/technical-stack.md

```diff
@@ -16,6 +16,18 @@ Not familiar with the Akeneo PIM REST API? We have a [getting started guide](/ge
 
 Also, if you are a PHP developer, we provide a [PHP Client](https://github.com/akeneo/api-php-client-ee) for this REST API. So don't hesitate to use it if need be!
 
+## What about the PIM Events API (Webhook)?
+
+If you have looked at our API documentation, you may have noticed that PIM also has a webhook mechanism called "Events API".
```

Comment:
maybe we should add something like: "you may have notices that since the 5.0, Akeneo PIM also has..."

Reply:
Good idea!